### PR TITLE
Update rector / php-cs-fixer config for test cases

### PR DESCRIPTION
**Pull request by core23**
Update rector / php-cs-fixer config for test cases.

POC: https://github.com/sonata-project/SonataAdminBundle/pull/8111

### CODE REVIEW

**Review by VincentLanglet — APPROVED** [review]

File: rector.php [hunk 1]
```diff
@@ -24,12 +26,17 @@
 
     $rectorConfig->sets([
         LevelSetList::UP_TO_PHP_74,
+        PHPUnitSetList::PHPUNIT_90,
+        PHPUnitSetList::PHPUNIT_91,
```
Comment (OskarStark) [inline rector.php:30]: Isn't there a `UP_TO_*` ?

Comment (core23) [reply rector.php:30]: Good catch! 

https://github.com/rectorphp/rector-phpunit/blob/main/src/Set/PHPUnitLevelSetList.php

I'll update this later

**Review by VincentLanglet — APPROVED** [review]
The composer normalize is failing if you have time to fix it

**Comment by core23** [comment]
Can we merge this? We might need some manual adjustments after dev-kit ran, because the phpunit data provider return type will be changed from `array` to `iterable`

**Review by VincentLanglet — APPROVED** [review]
it's ok to me

**Pull request merged**